### PR TITLE
Add V2EX Viewer extension

### PR DESCRIPTION
**Pull request by ghost**
## Description

A better extension for viewing V2EX topics and replies. Support two kinds of topic details display style, support to add proxy for access V2EX.

### Thumbnail view
v2ex-viewer-1

### Detailed view [V2EX Style]
v2ex-viewer-2

### Detailed view [Raycast Style]
v2ex-viewer-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/v2ex-viewer/src/components/TopicListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <List.Item
+      icon={member.avatar_mini}
+      title={topic.title}
+      subtitle={!isShowingDetail ? node.title : undefined}
+      accessories={
+        !isShowingDetail
+          ? [
+              {
+                tooltip: "主题回复数",
```
Comment (pernielsentikaer) [inline extensions/v2ex-viewer/src/components/TopicListItem.tsx:35]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/v2ex-viewer/src/components/TopicListItemDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <List.Item.Detail
+      markdown={getTopicMarkdownContent(props.topic, replies, topicDetailStyle === TopicDetailStyle.V2EX)}
+      metadata={
+        topicDetailStyle === TopicDetailStyle.Raycast && (
+          <List.Item.Detail.Metadata>
+            <List.Item.Detail.Metadata.Label title="主题相关信息" />
+            <List.Item.Detail.Metadata.Label title="标题" text={topic.title} />
+            <List.Item.Detail.Metadata.Label title="节点" icon={node.avatar_mini} text={node.title} />
+            <List.Item.Detail.Metadata.Label title="发布者" icon={member.avatar_mini} text={member.username} />
+            <List.Item.Detail.Metadata.Label title="发布时间" text={getUnixFromNow(topic.created)} />
```
Comment (pernielsentikaer) [inline extensions/v2ex-viewer/src/components/TopicListItemDetail.tsx:38]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/v2ex-viewer/src/utils/markdown.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const OP = (isOP: boolean) => {
+  return isOP ? Code("OP") : "";
+};
+
+const getTopicMarkdownContent = (topic: Topic, replies: Reply[], includeHeader = false) => {
+  const topicTitle = `${Heading(1, topic.title)}`;
+  const topicMember = `${Bold(Link(topic.member.username, topic.member.url))} · ${getUnixFromNow(topic.created)} · ${
+    topic.replies
+  } 条回复`;
```
Comment (pernielsentikaer) [inline extensions/v2ex-viewer/src/utils/markdown.ts:24]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

I left a few minor corrections for you, can you look into this?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by daikaiyu** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I left a few minor corrections for you, can you look into this?
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Hello! I have removed localized content and added new command. Looking forward to your review and suggestions again.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**